File: MqttPublish.js
const core = require('@huaweicloud/huaweicloud-sdk-core');
const iotda = require("@huaweicloud/huaweicloud-sdk-iotda");
const ak = "<ak>";
const sk = "<sk>";

const endpoint = "https://iotda.cn-north-4.myhuaweicloud.com";
const project_id = "<project id>";

const credentials = new core.BasicCredentials()
                     .withAk(ak)
                     .withSk(sk)
                     .withProjectId(project_id)
const client = iotda.IoTDAClient.newBuilder()
                            .withCredential(credentials)
                            .withEndpoint(endpoint)
                            .build();
const request = new iotda.CreateCommandRequest();
request.deviceId = "xyzer";
const body = new iotda.DeviceCommandRequest();
body.withParas({"time":{"data":Date.now()}});
body.withCommandName("posttime");
request.withBody(body);
const result = client.createCommand(request);
result.then(result => {
    console.log("JSON.stringify(result)::" + JSON.stringify(result));
}).catch(ex => {
    console.log("exception:" + JSON.stringify(ex));
});

File: mqttPublish.py

# coding: utf-8
import numpy as np
import time
from huaweicloudsdkcore.auth.credentials import BasicCredentials
from huaweicloudsdkiotda.v5.region.iotda_region import IoTDARegion
from huaweicloudsdkcore.exceptions import exceptions
from huaweicloudsdkiotda.v5 import *

if __name__ == "__main__":
    ak = "<ak>"
    sk = "<sk>"
    project_id = "<project id>"


    credentials = BasicCredentials(ak, sk) \

    client = IoTDAClient.new_builder() \
        .with_credentials(credentials) \
        .with_region(IoTDARegion.value_of("cn-north-4")) \
        .build()
    time_array = []
    for i in range(100):
        if i%10==0:
            time.sleep(4)
        try:
            request = CreateCommandRequest()
            request.device_id = "xyzer"
            request.body = DeviceCommandRequest(
                paras={"time":{"data":time.time()*1000}},
                command_name="posttime"
            )
            response = client.create_command(request)
            time_array.append(int(response.response['time']))
            print(f"花费的时间: {response.response['time']} ms")
        except exceptions.ClientRequestException as e:
            print(e.status_code)
            print(e.request_id)
            print(e.error_code)
            print(e.error_msg)
    if len(time_array)!=0:
        print(f"请求的次数：{len(time_array)},花费的平均时间：{sum(time_array)/len(time_array)} ms; 方差为{np.var(time_array)}")